oqs-sys/build.rs

```diff
@@ -117,7 +114,25 @@ fn main() {
     }
     println!("cargo:rustc-link-search=native={}", libdir.display());
 
-    let gen_bindings = |file, filter| generate_bindings(&outdir, file, filter);
+    outdir
+}
+
+fn main() {
+    // Check if clang is available before compiling anything.
+    bindgen::clang_version();
+
+    let config = pkg_config::Config::new()
+        .range_version("0.8.0".."0.9.0")
```

Comment:
Can we get this from the package version instead? The fewer places I need to update things...

Reply:
I think we could get the lower range bound from the crate version, but not sure how to handle upper bounds. With system packages that we don't have control over, it seems we should be flexible at least until the next incompatible version.

Reply:
Per semver, the next incompatible version from `0.8.x` is `0.9.0`. We should be able to simply decide this through "current minor version + 1"

Reply:
The problem is, we'd have to parse the version string if I understand correctly, and I don't know if we should do that manually, add some semver crate or sth else.

Reply:
Seems we can just use these environment variables:

```
CARGO_PKG_VERSION_MAJOR — The major version of your package.
CARGO_PKG_VERSION_MINOR — The minor version of your package.
CARGO_PKG_VERSION_PATCH — The patch version of your package.
```